src/scanpy/preprocessing/_pca/_dask.py

```diff
@@ -132,8 +132,7 @@ def transform_block(
             mean_=self.mean_,
             components_=self.components_,
             chunks=(x.chunks[0], self.n_components_),
-            meta=np.zeros([0], dtype=x.dtype),
-            dtype=x.dtype,
+            meta=np.zeros([]),
```

Comment:
```suggestion
            meta=np.zeros((), np.float64),
```

maybe we should be specific?